### PR TITLE
Migrated preferences title length test [Lint Test]kotlin

### PR DESCRIPTION
**Pull request by prateek-singh-3212**

### CODE REVIEW

File: lint-rules/src/test/java/com/ichi2/anki/lint/rules/FixedPreferencesTitleLengthTest.kt [hunk 1]
```diff
@@ -13,121 +13,126 @@
...
+            )
+            .issues(
+                FixedPreferencesTitleLength.ISSUE_MAX_LENGTH,
+                FixedPreferencesTitleLength.ISSUE_TITLE_LENGTH
+            )
+            .run()
+            .expectClean()
     }
 
+    companion object {
```
Comment (david-allison) [inline lint-rules/src/test/java/com/ichi2/anki/lint/rules/FixedPreferencesTitleLengthTest.kt:68]: move this back to the top of the file - makes the diff much easier to read and having it here adds no value

**Review by david-allison — CHANGES_REQUESTED** [review]
Thanks!

**Comment by prateek-singh-3212** [comment]
@mikehardy Sorry I chaged the value of string. Fixed Now

File: lint-rules/src/test/java/com/ichi2/anki/lint/rules/FixedPreferencesTitleLengthTest.kt (hunk 1 above)
Comment (prateek-singh-3212) [reply lint-rules/src/test/java/com/ichi2/anki/lint/rules/FixedPreferencesTitleLengthTest.kt:25]: done

**Review by lukstbit — APPROVED** [review]

**Pull request merged**